clang-tools-extra/include-cleaner/lib/FindHeaders.cpp

```diff
@@ -188,6 +290,20 @@ llvm::SmallVector<Hinted<Header>> findHeaders(const SymbolLocation &Loc,
     OptionalFileEntryRef FE = SM.getFileEntryRefForID(FID);
     if (!FE)
       return {};
+
+    if (SrcMgr::isSystem(
```

Comment:
i am afraid this check might be overly strict, maybe drop this initially and see if we have too many false-positives?